<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start Server</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Croonwolter&lt;/p&gt;&lt;p&gt;&amp;amp; Dros :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Control active</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="receiveControlLabel">
          <property name="text">
           <string>Unknown</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>SoC setpoint</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="Line" name="line_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="receiveSetpointLabel">
          <property name="text">
           <string>Unknown</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="Line" name="line_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="Line" name="line_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Poort nummer:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Car</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="portEdit">
        <property name="text">
         <string>8426</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="Line" name="line_11">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_13">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="Line" name="line_16">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Line" name="line_12">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="Line" name="line_14">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="Line" name="line_15">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="connectedLabel">
        <property name="styleSheet">
         <string notr="true">color: red</string>
        </property>
        <property name="text">
         <string>Disconnected</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="coupledTrueRadioButton">
             <property name="text">
              <string>True</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="coupledFalseRadioButton">
             <property name="text">
              <string>False</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Coupled</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="Line" name="line_17">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="localControlTrueButton">
             <property name="text">
              <string>True</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="localControlFalseButton">
             <property name="text">
              <string>False</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>State of Charge</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="Line" name="line_19">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Local Control</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="actualPowerEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Actual Power</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="Line" name="line_22">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Line" name="line_20">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>SoC setpoint</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="Line" name="line_23">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="carSoCEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="Line" name="line_24">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Line" name="line_21">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="Line" name="line_18">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="Line" name="line_25">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="Line" name="line_26">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="Line" name="line_27">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="Line" name="line_28">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="Line" name="line_29">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="currentSetpointLabel">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
